model/metrics/container.yaml

```diff
@@ -0,0 +1,72 @@
+groups:
+  # container.cpu.* metrics and attribute group
+  - id: attributes.container.cpu
+    prefix: container.cpu
+    type: attribute_group
+    brief: "Describes Container CPU metric attributes"
+    attributes:
+      - id: state
+        type:
+          allow_custom_values: false
+          members:
```

Comment:
Are we able to encode more information about what these values actually mean? User, system, kernel could be interpreted in different ways. They also mean slightly different things between Linux and Windows, so that might need to be clarified too. The docker stats receiver has descriptions of what it means for docker.

Reply:
Good idea added that at [253306e](https://github.com/open-telemetry/semantic-conventions/pull/282/commits/253306eec0f388d7821aa9ad7c4acaaf32994f01).